tests/brevitas/nn/test_recurrent.py

```diff
@@ -156,6 +215,35 @@ def test_quant_rnn_fwd_call(
             return_quant_tensor=return_quant_tensor)
         assert m(inp) is not None
 
+    @pytest.mark.parametrize("batch_first", [True, False])
+    @pytest.mark.parametrize("bidirectional", [True, False, 'shared_input_hidden_weights'])
+    @pytest.mark.parametrize("num_layers", [1, 2])
+    @pytest.mark.parametrize("bias", [True, False])
+    @pytest.mark.parametrize("return_quant_tensor", [True, False])
+    @pytest.mark.skip("FIXME: many warnings due to __torch_function__")
```

Comment:
note I'm disabling this test for now since there are many __torch__function warnings that need fixing

Reply:
I would let this and the other test run anyway.
The warnings during tests are a good reminder to go back to fix them.